dojo/tools/kiuwan/parser.py

```diff
@@ -50,33 +52,55 @@ def get_findings(self, filename, test):
-                                        "**Code at line number** : " + row['Line text'] + "\n\n" + \
-                                        "**Normative** : " + row['Normative'] + "\n\n" + \
-                                        "**Rule code** : " + row['Rule code'] + "\n\n" + \
-                                        "**Status** : " + row['Status'] + "\n\n" + \
-                                        "**Source file** : " + row['Source file'] + "\n\n" + \
-                                        "**Source line number** : " + row['Source line number'] + "\n\n" + \
-                                        "**Code at sorce line number** : " + row['Source line text'] + "\n"
-
-            finding.title = findingdict['title']
-            finding.file_path = findingdict['file']
-            finding.line = findingdict['line_number']
-            finding.description = findingdict['description']
+            severityfilter.eval_column(row["Priority"])
+            findingdict["severity"] = severityfilter.severity
+            findingdict["title"] = row["Rule"]
+            findingdict["file"] = row["File"]
+            findingdict["line_number"] = row["Line number"]
+            findingdict["description"] = (
+                "**Vulnerability type** : "
+                + row["Vulnerability type"]
+                + "\n\n"
+                + "**CWE Scope** : "
+                + row["CWE Scope"]
+                + "\n\n"
+                + "**Line number** : "
+                + row["Line number"]
+                + "\n\n"
+                + "**Code at line number** : "
+                + row["Line text"]
+                + "\n\n"
+                + "**Normative** : "
+                + row["Normative"]
+                + "\n\n"
+                + "**Rule code** : "
+                + row["Rule code"]
+                + "\n\n"
+                + "**Status** : "
+                + row["Status"]
+                + "\n\n"
+                + "**Source file** : "
+                + row["Source file"]
+                + "\n\n"
+                + "**Source line number** : "
+                + row["Source line number"]
+                + "\n\n"
+                + "**Code at sorce line number** : "
+                + row["Source line text"]
+                + "\n"
+            )
+
+            finding.title = findingdict["title"]
+            finding.file_path = findingdict["file"]
+            finding.line = findingdict["line_number"]
+            finding.description = findingdict["description"]
             finding.references = "Not provided!"
             finding.mitigation = "Not provided!"
-            finding.severity = findingdict['severity']
+            finding.severity = findingdict["severity"]
             finding.static_finding = True
             try:
-                finding.cwe = int(row['CWE'])
-            except:
+                finding.cwe = int(row["CWE"])
+            except BaseException:
```

Comment:
```suggestion
            except Exception:
```

Reply:
Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8304/commits/3683282176e99b2e46e9ecd1bd825090dcab1191